src/formio/components/DateField.stories.js

```diff
@@ -38,5 +41,61 @@ export const DateField = {
   args: {
     key: 'date',
     label: 'Datum',
+    extraComponentProperties: {
+      format: 'dd-MM-yyyy',
+      placeholder: 'dd-mm-yyyy',
+      enableTime: false,
+      datePicker: {
+        minDate: null,
+        maxDate: null,
+      },
+    },
+  },
+  play: async ({canvasElement}) => {
+    const canvas = within(canvasElement);
+
+    const dateInput = canvas.getByRole('textbox');
+
+    userEvent.type(dateInput, '06-06-2006');
+    await expect(dateInput).toHaveDisplayValue('06-06-2006');
+
+    const error = canvas.queryByText('minDate');
+    await expect(error).toBeNull();
+    // This test succeeds, but the value is not displayed in storybook... Mystery
+  },
+};
+
+export const DateWithMinField = {
+  render: SingleFormioComponent,
+  args: {
+    key: 'date',
+    label: 'Datum > 08-09-2023',
+    extraComponentProperties: {
+      format: 'dd-MM-yyyy',
+      placeholder: 'dd-mm-yyyy',
+      enableTime: false,
+      datePicker: {
+        minDate: '2023-09-08',
+        maxDate: null,
+      },
+      customOptions: {
+        allowInvalidPreload: true,
+      },
+      validate: {
+        dateMinMax: true,
+      },
+    },
+  },
+  play: async ({canvasElement}) => {
+    const canvas = within(canvasElement);
+
+    const dateInput = canvas.getByRole('textbox');
+
+    userEvent.type(dateInput, '06-06-2006');
+    await expect(dateInput).toHaveDisplayValue('06-06-2006');
+
+    // TODO: I cannot get this to work. If you do it manually in storybook, it works... (it shows the error).
+    // const error = canvas.queryByText('minDate');
+    // await expect(error).not.toBeNull();
```

Comment:
could be because of the missing `await`?

Have you tried `const error = await canvas.findByText('minDate')`? Possibly wrap it in `await waitFor(...);`? Probably the answer is yes :P

Reply:
I tried, but I couldn't get it to work :(

Reply:
date pickers are evil :)